tests/test_views.py

```diff
@@ -35,6 +35,10 @@ class SimpleView(DispatchHookMixin, tables.SingleTableView):
     model = Region  # required for ListView
 
 
+class SimpleNoTableClassView(DispatchHookMixin, tables.SingleTableView):
```

Comment:
You don't seem to use the functionality `DispatchHookMixin` provides, if so, please remove.